### DISCUSSION TITLE
bug: Symlink config files not found

### DISCUSSION THREAD

**Discussion by rogeriofrsouza** [LazyVim/LazyVim#3676] [Q&A] [answered]
Today I edited all my dotfiles, using the `GNU stow` strategy with symlinks, including the `~/.config/nvim` directory.

However, I just noticed that LazyVim can't list my config files when pressing the `c` in dashboard or `leader f c` in which-key.

This is the output:
![image](https://github.com/LazyVim/LazyVim/assets/77423511/7fa760d7-f420-475f-824c-37b24893c008)

I'm using `NVIM v0.9.5` and `Linux Mint 21.3 x86_64`

### Steps To Reproduce

1. `mkdir ~/.config/lazytest`
2. `git clone https://github.com/LazyVim/starter ~/.config/lazytest`
3. `NVIM_APPNAME=lazytest nvim` and wait for the installation to complete
4. `mkdir ~/dotfiles/.config`
5. `cp -r ~/.config/lazytest ~/dotfiles/.config`
6. `cd ~/dotfiles && stow --adopt . && cd`
9. Restart Neovim with `NVIM_APPNAME=lazytest nvim`
10. Press `c` for Config or `leader f c`

**Comment by dpetka2001** [comment]
If you type in the command line `lua require("telescope.builtin").find_files({cwd = vim.fn.stdpath("config")})` does it show anything?

**Reply by rogeriofrsouza** [reply]
It shows the same empty list, as in the image uploaded 😞

**Reply by dpetka2001** [reply]
Then it's a problem with Telescope that can't resolve linked files. You might want to open a feature request at their repo to support this, but there's nothing that can be done from LazyVim since it just uses the Telescope pickers as they come and nothing more.

**Reply by dpetka2001** [reply]
You might also want to take a look at [chezmoi](https://www.chezmoi.io/). This is what I use and it doesn't symlink the files and supports version control as well. There might be other tools I'm not aware of, that's why I suggested `chezmoi`.

**Reply by rogeriofrsouza** [reply]
Sure thing, I will check it out. Thank you!

**Reply by jeremyhofer** [reply, marked as answer]
Hello! I was just coming to report this here on LazyVim.

This is resolvable by setting `follow = true` in the options passed to `find_files`. By default telescope does not follow symlinks, but it may be enabled via `follow`.

Full example command that works: `lua require("telescope.builtin").find_files({cwd = vim.fn.stdpath("config"), follow = true})`

**Reply by folke** [reply]
ok, cool. Just made that the default for telescope

**Reply by brothert** [reply]
About symlinks in "Find Files" variants (now with Fzf):
```F1``` is very helpful and lists the ```Alt-f``` "follow symlinks" toggle.
